src/main/java/org/gentoo/java/ebuilder/maven/MavenEbuilder.java

```diff
@@ -301,7 +301,7 @@ private void writeDependencies(final Config config,
                             mavenProject.getCommonDependencies(), null);
                 }
 
-                if (!mavenProject.getTargetVersion().isEmpty()) {
+                if (mavenProject.getTargetVersion()!=null) {
```

Comment:
I see, so this line should be mavenProject.getTargetVersion()!=null && !mavenProject.getTargetVersion().isEmpty
to guard against null.